src/main/java/com/avispl/symphony/dal/infrastructure/management/tripleplay/TriplePlayAggregatorCommunicator.java

```diff
@@ -699,16 +750,40 @@ private List<String> getAllServiceOfClient(Client client) {
 
 	/**
 	 * get response of query client request
+	 *
 	 * @param macAddress list all mac address of clients
 	 * @param information list all information need to get
 	 * @return response of request
-	 * @throws Exception
 	 */
-	private String getResponseOfQueryClientRequest(List<String> macAddress,List<String> information) throws Exception {
+	private String getResponseOfQueryClientRequest(List<String> macAddress, List<String> information) throws Exception {
 		QueryClientRequestBody requestBody = new QueryClientRequestBody();
 		requestBody.setJsonrpc(TriplePlayConstrant.JSON_RPC);
 		requestBody.setClientMAC(macAddress);
 		requestBody.setInformation(information);
 		return doPost(TriplePlayURL.BASE_URI, requestBody.buildRequestBody());
 	}
+
+	/**
+	 * This method is used to handle input from adapter properties and convert it to Set of String for control
+	 *
+	 * @return Set<String> is the Set of String of filter element
+	 */
+	private Set<String> convertUserInput(String input) {
+		try {
+			if (!StringUtils.isNullOrEmpty(input)) {
+				String[] listAdapterPropertyElement = input.split(TriplePlayConstrant.COMMA);
+
+				// Remove start and end spaces of each adapterProperty
+				Set<String> setAdapterPropertiesElement = new HashSet<>();
+				for (String adapterPropertyElement : listAdapterPropertyElement) {
+					setAdapterPropertiesElement.add(adapterPropertyElement.trim());
+				}
+				return setAdapterPropertiesElement;
+			}
+			return Collections.emptySet();
+		} catch (Exception e) {
+			logger.error("In valid adapter properties input");
```

Comment:
Typo, "In valid" instead of "Invalid". Also, can you please make the value of input a part of error message, and also pass the original exception (e) to error()?

Reply:
I fixed it